api/core/tools/provider/builtin/duckduckgo/tools/ddgo_video.yaml

```diff
@@ -77,10 +80,18 @@ parameters:
         label:
           en_US: Long (>20 minutes)
           zh_Hans: 长视频（>20分钟）
-    label:
-      en_US: Video Duration
-      zh_Hans: 视频时长
     human_description:
       en_US: Filter videos by duration
       zh_Hans: 按时长筛选视频
     form: form
+  - name: proxy_url
+    label:
+      en_US: Proxy URL
+      zh_Hans: 视频代理地址
+    type: string
+    required: false
+    default: ""
+    human_description:
+      en_US: Please end with / , e.g. https://example.com/
```

Comment:
I think some other format proxy url is also avilable,  like `http://example.com/proxy?url=` ,  so we don't need to emphasize `end with /`